packages/cannon-worker-api/src/types.ts

```diff
@@ -301,6 +301,7 @@ export type WorldProps = {
   axisIndex: 0 | 1 | 2
   broadphase: Broadphase
   defaultContactMaterial: ContactMaterialOptions
+  frictionGravity?: Triplet
```

Comment:
```suggestion
  frictionGravity: Triplet | undefined
```
I feel like this is clearer as to what the acceptable values are.

Reply:
I'll take the suggestion, though note that these are two different typing statements: in the original, the parameter is optional, while in the suggestion, the parameter _must_ be provided but the value _may_ be `undefined`.

It won't make a difference to end users, since they configure the World via `CannonWorkerAPI` which is typed with `Partial<WorldProps>`, so everything becomes optional...but we will have to live with the knowledge of having lied to the TypeScript gods - I high price, but worth paying to keep the peace.